src/ethereum/ethNetwork.js

```diff
@@ -889,14 +1049,66 @@ export class EthereumNetwork {
     return response
   }
 
+  async checkTxsAmberdata(
+    startBlock: number,
+    currencyCode: string
+  ): Promise<EthereumNetworkUpdate> {
+    const address = this.ethEngine.walletLocalData.publicKey
+
+    let page = 0
+    const allTransactions: Array<EdgeTransaction> = []
+    while (1) {
+      const jsonObj = await this.fetchGetAmberdataApi(
```

Comment:
This is missing a way to query only the most recent transactions. Is this not available in the API? If it is not available then we will not implement this API. If it is available then please implement it

Reply:
Oh jeez, of course.  I completely forgot that.  I'll take a look later today.

Reply:
The Amberdata API  has a  "startDate" query param but it's the timestamp, not the blockNumber.  Is there a way to convert the starting blockheight to a timestamp?